src/SFA.DAS.EAS.Portal.Client/Models/Concrete/ReservedFundingDto.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using Newtonsoft.Json;
+
+namespace SFA.DAS.EAS.Portal.Client.Models.Concrete
+{
+    public class ReservedFundingDto : IReservedFundingDto
+    {
+        [JsonProperty("reservationId")]
+        public virtual Guid ReservationId { get; private set; }
```

Comment:
Should just be Id.  i.e. Reservation.Id .  No need for the tautology.

Reply:
Just Id is good.